xdr/signer_key.go

```diff
@@ -31,10 +41,10 @@ func (skey *SignerKey) Address() string {
 		key := skey.MustPreAuthTx()
 		copy(raw, key[:])
 	default:
-		panic(fmt.Errorf("Unknown signer key type: %v", skey.Type))
+		return "", fmt.Errorf("Unknown signer key type: %v", skey.Type)
```

Comment:
Error strings should start with a lower case variable because they get composed together separated by colons. See #2027.
```suggestion
		return "", fmt.Errorf("unknown signer key type: %v", skey.Type)
```